src/query/domain/QueryRequest.ts

```diff
@@ -0,0 +1,3 @@
+export class QueryRequest {
```

Comment:
JsDoc, please. We have so many interfaces/classes nowadays that I would love to have a one sentence summary of where each is exactly relevant.